node/src/components/contract_runtime.rs

```diff
@@ -429,6 +429,24 @@ impl ContractRuntime {
                 }
                 .ignore()
             }
+            ContractRuntimeRequest::GetTaggedValues {
+                request: tagged_values_request,
+                responder,
+            } => {
+                trace!(?tagged_values_request, "tagged values request");
+                let metrics = Arc::clone(&self.metrics);
+                let data_access_layer = Arc::clone(&self.data_access_layer);
+                async move {
+                    let start = Instant::now();
+                    let result = data_access_layer.tagged_values(tagged_values_request);
+                    metrics
+                        .get_all_values
+                        .observe(start.elapsed().as_secs_f64());
+                    trace!(?result, "get all values result");
```

Comment:
```suggestion
                    trace!(?result, "tagged values result");
```

for consistency with [this log](https://github.com/casper-network/casper-node/pull/4583/files#diff-5c13e80cf31df451ffa2cc0323731e9da3a1112109e65cde8ce6ad2fb56cf04fR436)